manifests/globals.pp

```diff
@@ -20,10 +20,7 @@
     'buster' => '4.4.8',
     default  => undef
   },
-  $manage_package_repo   = fact('os.distro.codename') ? { # Debian 10 doesn't provide mongodb packages. So manage it!
-    'buster' => true,
-    default  => undef
-  },
+  $manage_package_repo   = true,
```

Comment:
This seems like a breaking change to me and not sure what it contributes to this PR

Reply:
thanks @root-expert